### ISSUE TITLE
andd function for concur-core

### ISSUE THREAD

**Issue by seagreen** [ajnsit/concur#21]
It's mentioned in the README, and I see it [in the PureScript](https://pursuit.purescript.org/packages/purescript-concur-core/0.4.1/docs/Concur.Core.Types#v:andd) documentation, but I don't see one in `concur-core`. Would this be a good thing to add?

**Comment by ajnsit** [comment]
Yeah I should add it to concur-core as well. Feel free to open a PR, otherwise I will get around to it in a few days.
